CHANGELOG.rst

```diff
@@ -1,17 +1,71 @@
+  a project by using their reference DOI (Digital Object Identifier).
+- **UI**: Support declaring environment variables on session launchers.
+- **UI**: Legacy projects can now be migrated to Renku 2.0 directly from within Renku 2.0. (`#3613 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3613>`__)
+
+
+**✨ Improvements**
+
+- **UI**: Improved session launcher and environment views with clear build status indicators and separated launcher, environment, and session UX. (`#3648 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3648>`__)
+- **UI**: Enhanced session launcher and session pages with clearer actions, improved environment selector and added session share link. (`#3659 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3659>`__)
+
+**🐞 Bug Fixes**
+
+- **Core Service**: Fix a bug where removing activities wouldn't actually remove them.
+- **Data services**: Fix an issue where conflicting mount points would prevent sessions from starting.
+- **UI**: Fix an issue where pausing or resuming a session would crash the whole tab in Firefox.
+- **Gateway**: Do not crash if the metrics client is not enabled.
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+- **Helm chart**: Update the Keycloak theme image to use non-root user by default.
+- **Data services**: Added k8s cache service that caches sessions and builds in the data services database.
+- **Data services**: Added product metrics tracking.
+- **Gateway**: Added product metrics tracking.
+- **Data services**: Added data tasks deployment for running basic tasks in the scope of data services.
+- **Admin tools**: Add Harbor initialization script to setup a registry for RenkuLab v2.
+
+Individual Components
+~~~~~~~~~~~~~~~~~~~~~
+
+- `renku-data-services 0.39.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.39.0>`_
+- `renku-data-services 0.40.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.40.0>`_
+- `renku-data-services 0.41.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.41.0>`_
+- `renku-data-services 0.42.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.42.0>`_
+- `renku-data-services 0.43.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.43.0>`_
+- `renku-gateway 1.5.0 <https://github.com/SwissDataScienceCenter/renku-gateway/releases/tag/1.5.0>`_
+- `renku-gateway 1.5.1 <https://github.com/SwissDataScienceCenter/renku-gateway/releases/tag/1.5.1>`_
+- `renku-python 2.9.3 <https://github.com/SwissDataScienceCenter/renku-python/releases/tag/v2.9.3>`_
+- `renku-python 2.9.4 <https://github.com/SwissDataScienceCenter/renku-python/releases/tag/v2.9.4>`_
+- `amalthea-sessions 0.18.2 <https://github.com/SwissDataScienceCenter/amalthea/releases/tag/0.18.2>`_
+- `renku-ui 3.55.0 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.55.0>`_
```

Comment:
For completeness:

```suggestion
- `renku-ui 3.54.0 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.54.0>`_
- `renku-ui 3.55.0 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.55.0>`_
```

Reply:
I will accept this as soon as the renku-deploy in the CI pipeline finishes. Otherwise it will get stuck.